### PR TITLE
Add Verification Tests

### PR DESCRIPTION
**Pull request by arjun-io**
Very quick protocol configuration verification tests. We can expand on these as we deploy products

Some values are commented out as they haven't been set on mainnet yet. We'll update these values to the correct ones before merging this PR

### CODE REVIEW

**Comment by arjun-io** [comment]
@kbrizzle  this should finally be ready to review, contains all correct verification for Perennial v1.2

File: packages/perennial/test/verification/core/verifyCore.test.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  beforeEach(async () => {
+    const [signer] = await ethers.getSigners()
+    deployments = await HRE.deployments.all()
+    controller = Controller__factory.connect(deployments['Controller_Proxy'].address, signer)
+    collateral = Collateral__factory.connect(deployments['Collateral_Proxy'].address, signer)
+    incentivizer = Incentivizer__factory.connect(deployments['Incentivizer_Proxy'].address, signer)
+    proxyAdmin = ProxyAdmin__factory.connect(deployments['ProxyAdmin'].address, signer)
+    upgradeableBeacon = UpgradeableBeacon__factory.connect(deployments['UpgradeableBeacon'].address, signer)
+    timelock = TimelockController__factory.connect(deployments['TimelockController'].address, signer)
+    forwarder = Forwarder__factory.connect(deployments['Forwarder'].address, signer)
```
Comment (kbrizzle) [inline packages/perennial/test/verification/core/verifyCore.test.ts:47]: follow up: we can now safely remove forwarder from the codebase I think -- not in this PR though

File: packages/perennial/test/verification/core/verifyCore.test.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      expect(await timelock.getMinDelay()).to.equal(172800) // 48 hours
+      const timelockAdminRole = await timelock.TIMELOCK_ADMIN_ROLE()
+      const timelockProposerRole = await timelock.PROPOSER_ROLE()
+      const timelockExecutorRole = await timelock.EXECUTOR_ROLE()
+      const eqMultisigAddress = '0x589CDCf60aea6B961720214e80b713eB66B89A4d'
+      const pnlLabsMultisigAddress = '0xA20ea565cD799e01A86548af5a2929EB7c767fC9'
+      const deployerSigAddress = '0x66a7fDB96C583c59597de16d8b2B989231415339'
+
+      expect(await timelock.hasRole(timelockAdminRole, timelock.address)).to.be.true
+      expect(await timelock.hasRole(timelockAdminRole, getMultisigAddress('mainnet'))).to.be.true /* false */
```
Comment (kbrizzle) [inline packages/perennial/test/verification/core/verifyCore.test.ts:170]: nit: comment?

File: packages/perennial/test/verification/core/verifyCore.test.ts [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    })
+  })
+
+  /* describe('forwarder', () => {
+    it('has the correct configuration', async () => {
+      expect(await forwarder.DSU()).to.equal(deployments['DSU'].address)
+      expect(await forwarder.USDC()).to.equal(deployments['USDC'].address)
+      expect(await forwarder.batcher()).to.equal(deployments['Batcher'].address)
+    })
+  }) */
```
Comment (kbrizzle) [inline packages/perennial/test/verification/core/verifyCore.test.ts:198]: nit: comment?

**Review by kbrizzle — APPROVED** [review]

**Pull request merged**